This patch fixes the htaccess and config of univention-dojo-api-viewer (Bug #23197).
Please mind that univention-dojo-api-viewer/config.php index.php and lib/spider.php has CRLF lineendings!
Index: univention-dojo-api-viewer/config.php
===================================================================
--- univention-dojo-api-viewer/config.php	(Revision 804)
+++ univention-dojo-api-viewer/config.php	(Arbeitskopie)
@@ -3,22 +3,22 @@
 # basePath should represent the path from the DOCUMENT_ROOT
 # NOTE: to have the permalinks/REST-ful resource urls work, 
 # you'll need to make similar changes to the rewrite rules
-$basePath = "/";
+$basePath = "/univention-management-console-doc/";
 
-//$_base_url = "http://" . $_SERVER["HTTP_HOST"] . $basePath;
-$_base_url = "";
-$_site_name = "The Dojo Toolkit";
-$defVersion = "1.4";
-$dataDir = dirname(__FILE__) . "/data/";
+$_base_url = "http://" . $_SERVER["HTTP_HOST"] . $basePath;
+$_site_name = 'Univention Management Console Frontend API Documentation';
+$defVersion = "UMC";
+$dataDir = "/var/www/univention-management-console-doc/data/";
 $defPage = "";
 $default_theme = "dtk";
 $theme = "dtk";
 $filter_privates = true;
 $use_cache = true;
 $modules = array(
+	"umc"=>-1,
 	"dojo"=>-1, 
 	"dijit"=>-1, 
 	"dojox"=>-1, 
-	"djConfig"=>-1
+#	"djConfig"=>-1,
  );
 ?>
Index: univention-dojo-api-viewer/index.php
===================================================================
--- univention-dojo-api-viewer/index.php	(Revision 804)
+++ univention-dojo-api-viewer/index.php	(Arbeitskopie)
@@ -75,7 +75,7 @@
 <link rel="stylesheet" href="<?php echo $basePath; ?>themes/<?php echo $th; ?>/<?php echo $th; ?>.css" type="text/css" media="all" />
 <?php } ?>
 		<script type="text/javascript">djConfig={isDebug:false};</script>
-		<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/dojo/1.5/dojo/dojo.xd.js"></script>
+		<script type="text/javascript" src="/univention-dojo/dojo/dojo.js"></script>
 		<!-- SyntaxHighlighter -->
 		<script type="text/javascript" src="<?php echo $basePath; ?>js/syntaxhighlighter/scripts/shCore.js"><</script>
 		<script type="text/javascript" src="<?php echo $basePath; ?>js/syntaxhighlighter/scripts/shBrushJScript.js"><</script>
Index: univention-dojo-api-viewer/lib/transform.php
===================================================================
--- univention-dojo-api-viewer/lib/transform.php	(Revision 804)
+++ univention-dojo-api-viewer/lib/transform.php	(Arbeitskopie)
@@ -5,7 +5,7 @@
  *
  *
  */
-if (isset($_SERVER['HTTP_HOST'])) {
+if (PHP_SAPI !== 'cli') {
   die('Run from command line');
 }
 
Index: univention-dojo-api-viewer/lib/spider.php
===================================================================
--- univention-dojo-api-viewer/lib/spider.php	(Revision 804)
+++ univention-dojo-api-viewer/lib/spider.php	(Arbeitskopie)
@@ -38,7 +38,7 @@
  *	<!-- NAVIGATION -->
  */
 
-if(isset($_SERVER['HTTP_HOST'])) {
+if(PHP_SAPI !== 'cli') {
   die('Run from command line');
 }
 
Index: univention-dojo-api-viewer/.htaccess
===================================================================
--- univention-dojo-api-viewer/.htaccess	(Revision 804)
+++ univention-dojo-api-viewer/.htaccess	(Arbeitskopie)
@@ -1,14 +1,7 @@
-# supress php errors
-php_flag display_startup_errors off
-php_flag display_errors on
-php_flag html_errors off
-
-Options -MultiViews
-
 RewriteEngine On
 
 #  Set the prefix for all matches
-RewriteBase   /
+RewriteBase   /univention-management-console-doc/
 
 # convert static/rest-like requests
 RewriteRule ^html/([0-9.]+)/(.*)\.html$ lib/item.php?v=$1&p=$2 [L,QSA]
